src/main/resources/META-INF/plugin.xml

```diff
@@ -36,7 +36,7 @@
         ]]>
     </change-notes>
```

Comment:
Can you please also update minor version number and changelog entry to reflect your changes? It will make it ready for release. Thanks!